.github/workflows/tests.yml

```
@@ -37,6 +37,15 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Setup Cache
```

One huge nit, for consistency with your other changes.
```suggestion
    - name: Setup cache
```